docs/PublishingGuide.md

```diff
@@ -23,6 +22,20 @@
 It takes a few minutes for report generation. Meanwhile, you can monitor the progress live at [Travis-CI's Builds](https://travis-ci.org/dashboard/builds). <br/>
 Try accessing your site again when a green tick appears beside your fork.
 
+### Granting write-access to RepoSense for publishing
+We recommmend use of [personal access token](https://github.blog/2013-05-16-personal-api-tokens/) for ease of setup and [deploy key](https://developer.github.com/v3/guides/managing-deploy-keys/#deploy-keys) for enhanced security.
+
+##### If you wish to use `personal access token`:
+1. Follow this [guide](https://help.github.com/articles/creating-a-personal-access-token-for-the-command-line/) and give only `public_repo` permission <br/>
+1. **Copy** the token for later use <br/>
+
+##### If you wish to use `deploy key`:
+1. Use `ssh-keygen` to create a public/private key pair without a passphrase <br/>
+i.e. `ssh-keygen -t ecdsa -b 521 -f id_reposense -q -N ""`
+1. Go to the [deploy key settings](../../../../publish-RepoSense/settings/keys) of your publish-RepoSense fork and create a new deploy key with the contents of `id_reposense.pub`
+1. **Copy** the base64 encoded content of the private key for later use <br/>
+i.e. `cat id_reposense | base64 -w 0`
```

Comment:
This 'later use' is something mentioned later in our documentation?

Reply:
Yes, *later* in the setup instructions.

In Step 7:
7. Under **Environment Variables**, name a variable as `GITHUB_TOKEN` or `GITHUB_DEPLOY_KEY` depending on your earlier choice and paste the content that was copied earlier to its value field; then click **Add** <br/>

Reply:
Got it 👍